source/particle/property/crystal_preferred_orientation.cc

```diff
@@ -1078,23 +1254,25 @@ namespace aspect
                                Patterns::List(Patterns::Anything()),
                                "This determines what minerals and fabrics or fabric selectors are used used for the LPO/CPO calculation. "
                                "The options are Olivine: Passive, A-fabric, Olivine: B-fabric, Olivine: C-fabric, Olivine: D-fabric, "
-                               "Olivine: E-fabric, Olivine: Karato 2008 or Enstatite. Passive sets all RRSS entries to the maximum. The "
+                               "Olivine: E-fabric, Olivine: Karato 2008 or Enstatite or CPX. Passive sets all RRSS entries to the maximum. The "
                                "Karato 2008 selector selects a fabric based on stress and water content as defined in "
                                "figure 4 of the Karato 2008 review paper (doi: 10.1146/annurev.earth.36.031207.124120).");
 
+            prm.declare_entry ("CPX RRSS", "1,5,5,1.5",
+                               Patterns::List(Patterns::Anything()),
+                               "The RRSS values for CPX (clinopyroxene), used in fabric calculations.");
```

Comment:
What is RRSS? And why are there several RRSS values? This needs to be explained here, because it is not common knowledge.

Reply:
But you also need to add this here for the parameter documentation. Not every user will read the header file, and only if you put the documentation into the parameter description it will appear in our manual and here: https://aspect.geodynamics.org/doc/parameter_view/parameters.xml